src/api/public/apidocs/paths/person_login_token.yaml

```diff
@@ -68,19 +68,19 @@ post:
       example: Rebuild gchz
     - in: query
       name: project
+      required: true
       schema:
         type: string
       description: |
-        Project name. Should be provided, together with the package query parameter,
-        to limit the token to a specific package.
+        Project name. Limits the token to a specific project.
```

Comment:
Hmm, I would say we should not mark the project and package parameters as required. You can create a global token without providing those parameters. The description was correct before I would say

Reply:
Agreed, the edit made before doesn't consider the ability to create token without parameters.